### ISSUE TITLE
ADD COLUMN IF NOT EXISTS support in emulator

### ISSUE THREAD

**Issue by HengzhiShao-ANZ** [GoogleCloudPlatform/cloud-spanner-emulator#125]
The following works on Google Cloud Spanner console but not in spanner emulator:
```
ALTER table TableName
ADD COLUMN IF NOT EXISTS ColumnName BOOL NOT NULL DEFAULT (true);
```

Getting `Syntax error on line _, column _: Encountered 'IF' while parsing: identifier failed to init spanner emulator`

**Comment by mohitg1580** [comment]
This has been fixed  as part of this - https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/issues/119

**Comment by HengzhiShao-ANZ** [comment]
> This has been fixed as part of this - #119

Great! I guess this issue can be closed now.
